test/jbuilder_template_test.rb

```diff
@@ -44,7 +44,7 @@ def initialize(id, name)
 ActionView::Template.register_template_handler :jbuilder, JbuilderHandler
 
 PARTIALS = {
-  "_partial.json.jbuilder"  => "foo ||= 'hello'; json.content foo",
```

Comment:
Removed to be able to reproduce the undefined method error. This caused me to add
```ruby
-      json.partial! "partial"
+      json.partial! "partial", foo: 'hello'
```
Happy to discuss if we'd rather go back to defining its own partial.